### ISSUE TITLE
PHP7 compatibility

### ISSUE THREAD

**Issue by bandanh** [macopedia/community#1]
Hi there,
your Extension seems to be incompatible to PHP7
```
PHP Warning: Declaration of Macopedia\Community\ViewHelpers\SameUserViewHelper::render($requestingUser, $requestedUser) should be compatible with TYPO3\CMS\Fluid\Core\ViewHelper\AbstractConditionViewHelper::render() in typo3conf/ext/community/Classes/ViewHelpers/SameUserViewHelper.php line 0 (More information)
```

**Comment by bandanh** [comment]
Passing the arguments with `registerArgument` fixed it for me:

SameUserViewHelper:
```
    public function initializeArguments()
    {
        parent::initializeArguments();
        $this->registerArgument('requestingUser', 'object', 'Requesting User.');
        $this->registerArgument('requestedUser', 'object', 'Requested User.');
    }

        
    public function render() {
        $requestingUser = $this->arguments['requestingUser'];
        $requestedUser = $this->arguments['requestedUser'];
        
        ...
```

**Comment by bandanh** [comment]
Same for the HasAccessToAlbumViewHelper:
```
    public function initializeArguments()
    {
        parent::initializeArguments();
        $this->registerArgument('album', 'object', 'Album Object.');
        $this->registerArgument('relation', 'object', 'Relation Object.');
        $this->registerArgument('requestingUser', 'object', 'Requesting User Object.');
    }


    public function render()
    {
        /** @var Album $album */
        $album = $this->arguments['album'];
        /** @var Relation $relation */
        $relation = $this->arguments['relation'];
        /** @var User $requestingUser */
        $requestingUser = $this->arguments['requestingUser'];

        ...
```

**Comment by tmotyl** [comment]
hi @bandanh thanks for your contribution.
can you please provide a pull request?

**Comment by sventendo** [comment]
I hijacked the issue and made a pull request. see #2 .

**Comment by tmotyl** [comment]
solved with pull request #2
